app/controllers/meetings/publish/notulen.js

```diff
@@ -37,14 +37,16 @@ export default class MeetingsPublishNotulenController extends Controller {
       yield Promise.all(versionedNotulens.map(async (notulen) => {
```

Comment:
we map over versionedNotulen but on every iteration all we do is overwrite the local state.
as I understand it we could then just do the logic on the last element of the array no?

not even that, it uses Promise.all which means its arbitrary which will be the final state, no?

Reply:
No, because there will be 2 versioned notulens 1 with signedResources and one with publishedResources

Reply:
We want to get both

Reply:
ah I see, fair enough